packages/zosfiles/__tests__/__system__/api/methods/hMigrate/HMigrate.system.test.ts

```diff
@@ -115,7 +139,7 @@ describe("Migrate Dataset", () => {
     describe("Failure Scenarios", () => {
         beforeEach(async () => {
             try {
-                await Create.dataSet(REAL_SESSION, CreateDataSetTypeEnum.DATA_SET_SEQUENTIAL, dataSet1);
+                await Create.dataSet(REAL_SESSION, CreateDataSetTypeEnum.DATA_SET_SEQUENTIAL, dataSet3);
```

Comment:
Thanks for using the `...FAIL...` dataset name.
In the future we can use this patter (relying on specific values in the test-data) to experiment with more failure scenarios. 👍